### PR TITLE
Updated conditonal for caseString

### PR DESCRIPTION
**Pull request by crc8109**
Now will format cases depending if in millions or thousands

### CODE REVIEW

File: src/lib/coronavirus.js [hunk 1]
```diff
@@ -98,7 +98,9 @@ export function trackerFeatureToHtmlMarker({ properties = {} } = {}) {
 
     if ( stat?.type === 'number' ) {
       newValue = commafy(value);
-      if ( value > 999 ) {
+    if ( value > 999999 ) {
+        newValue = `${newValue.slice(0, -7)}m+
```
[the remark this answers is not in the document]
Comment (crc8109) [reply src/lib/coronavirus.js:102]: Oh snap good catch. I'll update

File: src/lib/coronavirus.js [hunk 2]
```diff
@@ -98,7 +98,9 @@ export function trackerFeatureToHtmlMarker({ properties = {} } = {}) {
...
+    } else if ( cases > 999 ) {
```
[the remark this answers is not in the document]
Comment (crc8109) [reply src/lib/coronavirus.js:103]: This was totally a typo. Apologies. I'll fix this too

**Comment by colbyfayock** [comment]
@crc8109  looks like something funky is still going on, check out the comma below. does the string need to be sliced at 8 instead of 7? 

image

If you're not familiar with Netlify here, you can check this out for yourself clicking this:
image

**Comment by crc8109** [comment]
That's strange. I'll take a look and come back with a solution

**Comment by colbyfayock** [comment]
Looks good. Thanks again

**Comment by crc8109** [comment]
Anytime!

**Pull request merged**